helpers/api_helpers.py

```diff
@@ -0,0 +1,14 @@
+"""
+This module defines helper functions for common operations in the Flask app.
+These functions help with generating consistent JSON responses.
+"""
+from flask import jsonify
+
+
+def create_response(status, message=None, data=None):
+    response = {"status": status}
+    if message:
+        response["message"] = message
+    if data:
+        response["data"] = data
+    return jsonify(response)
```

Comment:
You can even go further and save multiple of those 'error'/'success' strings

```suggestion
def create_succes_response(message=None, data=None):
    return create_response('sucess', message, data)

def create_error_response(message=None, data=None):
    return create_response('error', message, data)


def create_response(status, message=None, data=None):
    response = {"status": status}
    if message:
        response["message"] = message
    if data:
        response["data"] = data
    return jsonify(response)
```

Reply:
Great suggestion! Thanks!